packages/react-components/src/Concepts/Positioning/PositioningOffset.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { PositionedComponent } from './utils.stories';
+
+export { Default } from './PositioningDefault.stories';
+export { ShorthandPositions } from './PositioningShorthandPositions.stories';
+export { CoverTarget } from './PositioningCoverTarget.stories';
+
+export const Offset = () => {
+  const [offsetY, setOffsetY] = React.useState(10);
+  const [offsetX, setOffsetX] = React.useState(10);
+
+  const onChangeY = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setOffsetY(parseInt(e.target.value, 10));
+  };
+
+  const onChangeX = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setOffsetX(parseInt(e.target.value, 10));
+  };
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column', gap: 10 }}>
+      <label style={{ display: 'flex', gap: 10 }}>
+        Offset Y
+        <input onChange={onChangeY} value={offsetY} type="number" />
+      </label>
+      <label style={{ display: 'flex', gap: 10 }}>
+        Offset X
+        <input onChange={onChangeX} value={offsetX} type="number" />
+      </label>
+      <PositionedComponent
+        positioning={{ position: 'after', offset: [offsetY, offsetX] }}
+        targetContent="Simple offset"
+      />
+      <PositionedComponent
+        positioning={{ position: 'after', offset: () => [offsetY, offsetX] }}
+        targetContent="Offset function"
+      />
```

Comment:
I suggest to move out the case with function into a separate example and show there how to deal with arguments:

```tsx
      <PositionedComponent
        positioning={{ position: 'after', offset: ({ popper, reference }) => [10, popper.width / 2] }}
        targetContent="Offset function"
      />
```